GOVERNANCE.md

```diff
@@ -124,11 +127,30 @@ the issue tracker is:
 
 ## Collaborator nominations
 
-Existing collaborators can nominate someone to become a collaborator. Nominees
-should have significant and valuable contributions across the Node.js
+### Who can nominate Collaborators?
+
+Existing Collaborators can nominate someone to become a Collaborator.
+
+### Ideal Nominees
+
+Nominees should have significant and valuable contributions across the Node.js
 organization.
 
-To nominate a new collaborator, open an issue in the [nodejs/node][] repository.
+Contributions can be:
+
+* Commits
+* Pull requests
+* Comments
+* Reviews
+* Helping others
+* Issues
+* Participation in other projects, teams, and working groups of the Node.js
+  organization
+* Other participation in the wider Node.js community
```

Comment:
I think this might be slightly outdated now that we also have a triager role. Contributors who primarily help others, triage issues, comment on PRs and issues, participate in some teams, etc., do not benefit from the commit bit, nor does the project improve their ability to contribute by making them a collaborator in that case.